app/views/invoices/_form.html.haml

```diff
@@ -66,9 +75,12 @@
             = label_tag "Send To "
             = label_tag " pragati.doshi@botreeconsulting.com"
           .box-footer
+            = hidden_field_tag :email_send, :value => "false", :id => "email_send"
             = f.submit "Save as Draft", class: "btn btn-primary", id: "submit_button", data: { disable_with: "Please wait..." }
-            = f.submit "Send by Email", class: "btn btn-primary", id: "send_button", data: { disable_with: "Please wait..." }
+            = f.button "Save & Send by Email", class: "btn btn-primary", id: "send_button", data: { disable_with: "Please wait..." }
+
 :javascript
+
   $(function() {
-    $('.datepicker').datepicker();
+    $('.datepicker').datepicker({ dateFormat: "yyyy-mm-dd", autoclose: true });
```

Comment:
Everywhere the date format is `%m%d%y`. Shouldn't it be consistent across the app?